src/transformers/dependency_versions_check.py

```diff
@@ -0,0 +1,11 @@
+from .dependency_versions_table import deps
+from .utils.versions import require_version_core
+
+
+# define which module versions we always want to check at run time (usually the ones defined in `install_requires` in setup.py)
+pkgs_to_check_at_runtime = "python tokenizers tqdm regex numpy".split()
+for k in pkgs_to_check_at_runtime:
+    if k in deps:
+        require_version_core(deps[k])
+    else:
+        raise ValueError(f"can't find {k} in {deps.keys()}, check dependency_versions_table.py")
```

Comment:
I think it would be nicer here to directly show the user which dependencies are required instead of making him look for another file => Can't we directly output. Please make sure to install xxx with `pip install ....`?

Reply:
Agreed!

Reply:
this is not for the user, but for a developer who added must-have-this-available at run-time but forgot to add it to dependence table. This exception shouldn't happen, but it's there to save grace for the developer.

Let me know if it makes sense now.

Reply:
Got it! Yeah this makes sense to me